import-service/lambda-handlers/importProductsFile.js

```diff
@@ -0,0 +1,31 @@
+import AWS from 'aws-sdk';
+import { BUCKET } from '../constants/bucket-constants';
+
+export const importProductsFile = async event => {
+    try{
+      const { queryStringParameters: { name }} = event;
+      const params = {
+        Bucket: BUCKET,
+        Key: `uploaded/${ name }`,
+        Expires: 60,
+        ContentType: 'application/vnd.ms-excel'
```

Comment:
It seem that it wouldn't work with csv files, which are created with another applications.
If other content types don't work for you try to add "signatureVersion: "v4" " to your AWS.S3 constructor

Reply:
FE app returns 403 when I try to upload csv file